features/step_definitions/web_steps.rb

```diff
@@ -265,6 +265,15 @@ def with_scope(locator, &block)
   expect(page).to have_button(text)
 end
 
+Then(/^(?:|I )should not see a button named "([^"]*)"$/) do |text|
+  expect(page).to have_no_button(text)
+end
+
 Given(/I seed data/) do
   Rails.application.load_seed
 end
+Then(/^"([^"]*)" should be selected for "([^"]*)"(?: within "([^"]*)")?$/) do |value, field, selector|
```

Comment:
```suggestion

Then(/^"([^"]*)" should be selected for "([^"]*)"(?: within "([^"]*)")?$/) do |value, field, selector|
```

Nit, spacing before a block of code.